### PR TITLE
adding runtime configuration for geth and postgres

### PR DESCRIPTION
**Pull request by aupiff**
New configuration fields in `lndr-server.config`:
```
+# postgres config
+postgresHost = "localhost"
+postgresPort = 5432
+
+# ethereum node config
+web3Host = "http://localhost"
+web3Port = 8545
```

### CODE REVIEW

File: lndr-backend/data/lndr-server.config.test [hunk 1]
```diff
@@ -35,3 +35,11 @@ heartbeatInterval = 5
 awsPhotoBucket = "lndr-avatars"
 accessKeyId = "AAAAAAAAAAAAAAAAAAAA"
 secretAccessKey= "6444444444444444444444444444444444444444"
+
+# postgres config
+postgresHost = "localhost"
+postgresPort = 5432
+
+# ethereum node config
+web3Host = "http://localhost"
```
Comment (canterberry) [inline lndr-backend/data/lndr-server.config.test]: The value of `web3Host` here appears to be a full URL. We have a couple of options here:

 1. Rename this to `web3Url` and incorporate the port as well, like `http://localhost:8545`, or
 2. Change the expected value to be, like `postgresHost`, a hostname or IP.

If we go the route of option 2, then we may also want a `web3Protocol` attribute that can be either `http` or `https`.

File: lndr-backend/lndr-backend.cabal [hunk 2]
```diff
@@ -25,13 +25,15 @@ library
                , Lndr.Db.PushData
                , Lndr.Db.Types
                , Lndr.Db.VerifiedCredits
+               , Lndr.Config
```
Comment (canterberry) [inline lndr-backend/lndr-backend.cabal]: **Nit:** It looks this list is currently sorted alphabetically, but `Lndr.Config` is being inserted here after `Lndr.Db.`. Should be the first entry in the `other-modules:` list.

File: lndr-backend/src/Lndr/Config.hs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                          (loadEntry "gasPrice")
+                          (loadEntry "maxGas")
+                          (loadEntry "urbanAirshipKey")
+                          (loadEntry "urbanAirshipSecret")
+                          (loadEntry "heartbeatInterval")
+                          (loadEntry "awsPhotoBucket")
+                          (loadEntry "accessKeyId")
+                          (loadEntry "secretAccessKey")
+                          (loadEntry "postgresHost")
+                          (loadEntry "postgresPort")
```
Comment (canterberry) [inline lndr-backend/src/Lndr/Config.hs]: **Non-blocking:** We're using the `db` prefix above for database-related configuration, but here we're using a `postgres` prefix. Should both sets of configuration values use the same prefix?

File: lndr-backend/src/Lndr/Types.hs [hunk 4]
```diff
@@ -224,6 +225,10 @@ data ServerConfig = ServerConfig { lndrUcacAddr          :: !Address
                                  , awsPhotoBucket        :: !Text
                                  , awsAccessKeyID        :: !ByteString
                                  , awsSecrtAccessKey     :: !ByteString
```
Comment (canterberry) [inline lndr-backend/src/Lndr/Types.hs]: `awsSecrtAccessKey`… is this intentional or typo?

Comment (aupiff) [reply lndr-backend/src/Lndr/Types.hs]: lol, typo, thanks. using too much VIM autocomplete.

File: lndr-backend/src/Lndr/Config.hs (hunk 3 above)
Comment (aupiff) [reply lndr-backend/src/Lndr/Config.hs]: right, this was sloppy!

File: lndr-backend/lndr-backend.cabal (hunk 2 above)
Comment (aupiff) [reply lndr-backend/lndr-backend.cabal]: right!

File: lndr-backend/data/lndr-server.config.test (hunk 1 above)
Comment (aupiff) [reply lndr-backend/data/lndr-server.config.test]: Good suggestion. Going with option 1.

**Pull request merged**